src/dmd/dmodule.d

```diff
@@ -571,6 +538,69 @@ extern (C++) final class Module : Package
         return m;
     }
 
+    /**
+     * Converts a chain of identifiers to the filename of the module
+     */
+    static const(char)* getFilename(Identifiers* packages, Identifier ident)
+    {
+        auto filename = ident.toChars();
+        if (packages && packages.dim)
```

Comment:
My dream: that one day all functions ending with a single long `if` are written to first eliminate the negation of that `if` as the uninteresting case, to then proceed with the interesting part without indentation.